### DISCUSSION TITLE
issue with flux connecting to Github, done  too many requests to Github so there is a "client rate limiter" applied from Github to stop denial of service attacks in my account

### DISCUSSION THREAD

**Discussion by ex-aswath-edx** [community/community#127645] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

issue with flux connecting to Github, done  too many requests to Github so there is a "client rate limiter" applied from Github to stop denial of service attacks in my account

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @aswath-edx ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
